packages/ubuntu_wsl_setup/lib/main_common.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:args/args.dart';

```

Comment:
nit-pick: files called "main" are expected to have a "main" entry-point. :wink: could we give this file a name without "main" in it, please? perhaps something related to cli or args?